app/features/badges/homemade.json

```diff
@@ -423,6 +423,18 @@
 		"displayName": "Reef Reunion (500 KT Power)",
 		"authorDiscordId": "1170249805373657093"
 	},
+	"lanchampfirst": {
+		"displayName": "LAN Championship Showdown",
+		"authorDiscordId": "201922904781357057"
+	},
+	"lanchampsecond": {
+		"displayName": "LAN Championship Showdown",
+		"authorDiscordId": "201922904781357057"
+	},
+	"lanchampthird": {
+		"displayName": "LAN Championship Showdown",
+		"authorDiscordId": "201922904781357057"
+	},
```

Comment:
The display name should not be the same for everything. It should be for example "LAN Championship Showdown (1st place)", "LAN Championship Showdown (2nd place") etc.